### DISCUSSION TITLE
Golang connection string incorrect

### DISCUSSION THREAD

**Discussion by gursheyss** [planetscale/discussion#554] [Bug Reports] [answered]
Hi, I was trying to setup a go connection with my server and noticed the connection string was wrong in the UI
![image](https://github.com/planetscale/discussion/assets/116788218/f51068bb-7220-4b68-9476-cb6743a51799)
Instead of being in the correct DSN format it had this weird formatting, After manually changing the string it worked but this one it gave was invalid

**Comment by mscoutermarsh** [comment, marked as answer]
@gursheyss Thanks for reporting, we've fixed it!
